.rubocop_local.yml

```diff
@@ -0,0 +1,3 @@
+AllCops:
+  Exclude:
+  - db/schema.rb
```

Comment:
We don't have a `db` directory in this repo

Reply:
ok, I copied it from other repo